src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -556,7 +606,7 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 		<dev:version />
 	</command:details>
 	<maml:description>
-		<maml:para>List all registered URLACls. The listed entries can be used to diagnose URLACL issues which may prevent ServiceControl from starting. If you require a more feature rich tool for managing URLACLs please take a look at NetSh.exe</maml:para>
+		<maml:para>List all registered URLACls. The listed entries can be used to diagnose URLACL issues which may prevent ServiceControl from starting. If you require a more feature rich tool for managing URLACLs please take a look at NETSH.EXE.</maml:para>
```

Comment:
Lower cased L again.

This one uses an all-uppercased NETSH but above it is still NetSh.exe. Is one of them wrong?